almond/Dockerfile

```diff
@@ -0,0 +1,45 @@
+ARG BUILD_FROM
+FROM ${BUILD_FROM}
+
+# Install Alomond requirements
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        git \
```

Comment:
`git` could be moved to the second RUN command, that will shave off a couple of MB's from the resulting image (since it can be removed at the end).